modules/migration-mtc-release-notes-1-7-8.adoc

```diff
@@ -0,0 +1,25 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-8_{context}"]
+= {mtc-full} 1.7.8 release notes
+
+[id="new-features-and-enhancements-1-7-8_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* Added configuration option to set 'spc_t' as SELinuxOptions Type in Rsync containers.
+
+* Added configuration option to override Velero images.
```

Comment:
@HagayVider1 This should be in the Resolved issues section.

"Fixed configuration option to override Velero image."

Reply:
@stillalearner 
Resolved. Please check.